ddtrace/profiling/collector/_utils.h

```diff
@@ -4,6 +4,15 @@
 #include <Python.h>
 #include <stdlib.h>
 
+
+static inline uint64_t
+random_range(uint64_t max)
+{
+    /* Return a random number between [0, max[ */
```

Comment:
```suggestion
    /* Return a random number between [0, max) */
```

Reply:
Is the backwards bracket a notation I'm not familiar with? 🤔 I see it used above as well

Looked it up: https://math.stackexchange.com/questions/653425/whats-the-difference-between-a-b-and-a-b

Reply:
It's old-fashioned maths notation. Last time I've used it I was a kid 🙂

Reply:
Haha, sorry. It's not old here, it's how we do it in France: https://fr.wikipedia.org/wiki/Intervalle_(math%C3%A9matiques) 

🤷

Reply:
Ah I see, this must be one of those ripples generated by the Anglo-French Wars that still goes around 😄 

> Une autre notation (d'origine anglaise mais très répandue également) utilise, pour les intervalles (semi-)ouverts, une parenthèse au lieu d'un crochet